### PR TITLE
Delete Booking Route

### PR DESCRIPTION
**Pull request by gekyxme**

### CODE REVIEW

File: routes/bookings/deleteBooking.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+	"github.com/gofiber/fiber/v2"
+)
+
+// DEETE /bookings/:id
+// DeleteBooking handles the deletion of a booking
+// Accepts a booking id as a url parameter
+// Returns a status code and a message
+func DeleteBooking(c *fiber.Ctx) error {
+	bookingId, err := c.ParamsInt("id")
```
Comment (RohanKhatua) [inline routes/bookings/deleteBooking.go:17]: check id using strconv and atoi

**Review by RohanKhatua — CHANGES_REQUESTED** [review]
Please see review comments

File: routes/bookings/deleteBooking.go (hunk 1 above)
Comment (gekyxme) [reply routes/bookings/deleteBooking.go]: Made the changes

**Review by RohanKhatua — APPROVED** [review]
LGTM

**Pull request merged**